main/src/cgeo/geocaching/connector/lc/LCApi.java

```diff
@@ -199,7 +203,9 @@ private static Geocache parseCacheDetail(final JsonNode response) {
             cache.setDifficulty((float) 1);
             cache.setTerrain((float) 1);
             cache.setSize(CacheSize.getById("virtual"));
-            cache.setFound(false);
+            if (DataStore.isThere(geocode + uuid, "", false)) {
```

Comment:
Don't `geocode + uuid` result in a string like "LC_uuid_uuid"? Seems like you are adding the `uuid` twice...

Reply:
Yes, that is without any doubts a bug introduced via careless copy / paste. Thx much.